### PR TITLE
Use CSS variables to handle theming (part 2)

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Follow-up of https://github.com/rust-lang/rust/pull/98460.

As usual, there shouldn't be any UI change.

You can test it [here](https://rustdoc.crud.net/imperio/css-theme-var/bar/index.html).

r? @jsha

### CODE REVIEW

File: src/librustdoc/html/static/css/themes/ayu.css [hunk 1]
```diff
@@ -9,6 +9,11 @@ Original by Dempfi (https://github.com/dempfi/ayu)
 	--settings-input-color: #ffb454;
 	--sidebar-background-color: #14191f;
 	--sidebar-background-color-hover: rgba(70, 70, 70, 0.33);
+	--code-block-background-color: #191f26;
+	--scrollbar-track-background-color: transparent;
+	--scrollbar-thumb-background-color: #5c6773;
+	--scrollbar-color: #5c6773 #24292f;
```
Comment (jsha) [inline src/librustdoc/html/static/css/themes/ayu.css:15]: This shows an interesting inconsistency. I would expect `transparent` to be `#24292f` or vice verse. Any idea why there's a difference?

**Review by jsha — APPROVED** [review]

File: src/librustdoc/html/static/css/themes/ayu.css (hunk 1 above)
Comment (GuillaumeGomez) [reply src/librustdoc/html/static/css/themes/ayu.css:15]: > I would expect `transparent` to be `#24292f` or vice verse.

Sorry I'm a bit lost on this part. Could you explain a bit more please? 😅

Comment (GuillaumeGomez) [reply src/librustdoc/html/static/css/themes/ayu.css:15]: Oh I just understood. I'm not too sure to be honest. It seems a bit weird indeed.

**Pull request merged**